### PR TITLE
Fix `MQCNN` ignoring zero-seed. (#2378)

### LINKED ISSUE #2378: Bug using zero as seed to MQCNNEstimator
## Description
(A clear and concise description of what the bug is.)
For the MQCNNEstimator, when the seed is set to 0 the results do not reproduce between two runs. If the seed is set to 1 they do. This can be seen from the `avg_epoch_loss` printouts. 

## To Reproduce
(Please provide minimal example of code snippet that reproduces the error. For existing examples, please provide link.)

```python
from gluonts.mx.model.seq2seq import MQCNNEstimator
from gluonts.mx import Trainer
from gluonts.dataset.repository.datasets import get_dataset

dataset = get_dataset('electricity')
dataset_list = list(dataset.train)
train_data = dataset_list[0]

seed = 1 # Or 0

for ii in range(2): # Run twice    
    print('%sth run:' %ii)

    trainer = Trainer(epochs=2)
    mq_cnn = MQCNNEstimator(prediction_length=24, 
                            freq='H', 
                            trainer=trainer,
                            seed=seed) 
    model = mq_cnn.train(training_data=[train_data])
```

## Error message or code output
(Paste the complete error message, including stack trace, or the undesired output that the above snippet produces.)

Seeded with 0:
```
0th run:

100%|██████████████████████████████████| 50/50 [00:12<00:00,  3.98it/s, epoch=1/2, avg_epoch_loss=0.484]
100%|████████████████████████████████████| 50/50 [00:12<00:00,  4.02it/s, epoch=2/2, avg_epoch_loss=0.3]

1th run:

100%|██████████████████████████████████| 50/50 [00:12<00:00,  4.01it/s, epoch=1/2, avg_epoch_loss=0.529]
100%|██████████████████████████████████| 50/50 [00:12<00:00,  4.06it/s, epoch=2/2, avg_epoch_loss=0.327]

```

Seeded with 1:
```
0th run:

100%|██████████████████████████████████| 50/50 [00:12<00:00,  3.99it/s, epoch=1/2, avg_epoch_loss=0.469]
100%|██████████████████████████████████| 50/50 [00:12<00:00,  3.99it/s, epoch=2/2, avg_epoch_loss=0.276]

1th run:

100%|██████████████████████████████████| 50/50 [00:12<00:00,  4.00it/s, epoch=1/2, avg_epoch_loss=0.469]
100%|██████████████████████████████████| 50/50 [00:12<00:00,  4.01it/s, epoch=2/2, avg_epoch_loss=0.276]
```

## Environment
- Operating system: Mac OS Monterey
- Python version: 3.8.13
- GluonTS version: 0.10.6
- MXNet version: 1.9.1

### PR DESCRIPTION
**Pull request by sighellan**
Fixes: https://github.com/awslabs/gluonts/issues/2378

*Description of changes:*
- Fix to how we check whether a seed has been given as input so integer 0 is also used. 
- Small typo fix.

Seeded with 0 output is now:
```
0th run:

100%|██████████████████████████████████| 50/50 [00:12<00:00,  4.00it/s, epoch=1/2, avg_epoch_loss=0.484]
100%|████████████████████████████████████| 50/50 [00:12<00:00,  4.00it/s, epoch=2/2, avg_epoch_loss=0.3]

1th run:

100%|██████████████████████████████████| 50/50 [00:12<00:00,  4.01it/s, epoch=1/2, avg_epoch_loss=0.484]
100%|████████████████████████████████████| 50/50 [00:12<00:00,  4.04it/s, epoch=2/2, avg_epoch_loss=0.3]
```
By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

**Please tag this pr with at least one of these labels to make our release process faster:** bug fix

### CODE REVIEW

**Review by jaheba — APPROVED** [review]
Good spot!
